peer_mentoring/groups/models.py

```diff
@@ -38,6 +39,22 @@ def __str__(self):
         return self.title
 
 
+class GroupJoinRequest(models.Model):
```

Comment:
merge today's PR into this one, it has some fixes to this, then merge all to main and start a new branch please